### PR TITLE
update catalog metadata

### PR DESCRIPTION
**Pull request by fightingmonk**

### CODE REVIEW

File: catalog/config.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "address": "opensource+nr1-tag-improver@newrelic.com"
     },
     "community": {
-      "url": "https://discuss.newrelic.com/t/[YOUR REPO SUPPORT TOPIC]"
+      "url": "https://discuss.newrelic.com/t/tag-improver-new-relic-one-nerdpack/125683"
     }
   },
   "whatsNew": "\n-Fix: One change that was made\n-Fix: Another change that was made\n-Feat: One feature that was added"
```
Comment (JuliaNocera) [inline catalog/config.json:16]: I would just remove this if you don't want to add some what's new type items ;)

File: catalog/config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "tagline": "[30 characters]",
+  "tagline": "Get your entity tagging sorted",
   "repository": "https://github.com/newrelic/nr1-tag-improver.git",
   "details": "[]",
```
Comment (JuliaNocera) [inline catalog/config.json:4]: Is there a "long form" version of the tagline you might be able to add here? This is what will populate the main details on the initial landing page for app info in the catalog.

Comment (JuliaNocera) [reply catalog/config.json:4]: Here's a bit more of a visual breakdown of what the config maps to
New_Relic_One_catalog___New_Relic_One

**Comment by fightingmonk** [comment]
Thanks for explaining the catalog schema @JuliaNocera - I've made some updates that should get us closer to presentable.

**Review by JuliaNocera — APPROVED** [review]
Looks good!

**Review by Robin-Law — APPROVED** [review]

**Pull request merged**